### ISSUE TITLE
bb-grid: Filter button missing text equivalent in demo (1.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#934]
### Expected behavior
Filter button has text equivalent. 

### Actual behavior

The button is missing the text equivalent in the first example but not the second. It's also ok in the filter component so thinking this is demo issue, but not sure how it's possible to use the filter component and not get the title attribute with it?
